Dockerfile.tableau

```diff
@@ -0,0 +1,23 @@
+FROM node:22-alpine AS builder
+WORKDIR /app
+
+COPY package.json package-lock.json tsconfig.json ./
```

Comment:
I think we can just close the repo once here.

Reply:
Sorry, yes. Clone the repo.

Reply:
Fair point. Updated